src/app/space-settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -4,6 +4,10 @@ import { Modal } from 'ngx-modal';
 import { Context, AreaService, Area, AreaAttributes } from 'ngx-fabric8-wit';
 import { ContextService } from '../../../shared/context.service';
 
+interface Errors {
+  unique: boolean
```

Comment:
This interface should be pulled out into the models folder.  Can you also change the name of unique to uniqueValidationFailure?  This makes it sound like it's failing because it's unique and not the other way around.